ADAL/src/public/ADUserInformation.h

```diff
@@ -34,15 +34,21 @@
 }
 
 /*! Factory method to extract user information from the AAD id_token parameter.
- @param idToken The contents of the id_token parameter, as passed by the server. */
-+ (ADUserInformation *) userInformationWithIdToken:(NSString *)idToken
-                                             error:(ADAuthenticationError * __autoreleasing *)error;
+ @param idToken The contents of the id_token parameter, as passed by the server.
+ @param homeUserId Combination of uid & utid.
+ */
++ (ADUserInformation *)userInformationWithIdToken:(NSString *)idToken
+                                       homeUserId:(NSString *)homeUserId
+                                            error:(ADAuthenticationError * __autoreleasing *)error;
 
 /* This is the only unique property, as it is used in the key generation for the cache.
  Two ADUserInformation objects are considered the same if this property is the same. Using RequiredDisplayableId
  will validate against this property. */
 @property (readonly) NSString* userId;
 
+/*! Combination of uid & utid. */
```

Comment:
Please provide more information about homeUserId - that it's a unique AAD user identifier across tenants based on user's home OID/home tenantId.

Reply:
done